Modules/_tkinter.c

```diff
@@ -2667,10 +2667,7 @@ FileHandler(ClientData clientData, int mask)
     func = data->func;
     file = data->file;
 
-    arg = Py_BuildValue("(Oi)", file, (long) mask);
```

Comment:
>  Additionally, argument for "i" format should be int, not (long).

Right, I spotted it as well, but then I forgot to report it, sorry :-) Hopefully you fixed it!